src/SolutionTemplate/UnoSolutionTemplate.WinUI.netcore/Views/UnoQuickStart.csproj

```
@@ -0,0 +1,36 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+	<PropertyGroup>
+		<TargetFrameworks>$libarybasetargetframework$</TargetFrameworks>
+		$if$($UseWinUI$ == True)<TargetFrameworks>$basetargetframework$-windows10.0.18362</TargetFrameworks>$endif$
```

this doesn't include the $(TargetFrameworks) and will result in the `$librarybasetargetframework$` not getting included in the TargetFrameworks.
```suggestion
		$if$($UseWinUI$ == True)<TargetFrameworks>$(TargetFrameworks);$basetargetframework$-windows10.0.18362</TargetFrameworks>$endif$
```

---

Indeed, I shuffled the lines and forgot that one, thanks!